pyvista/utilities/helpers.py

```diff
@@ -14,10 +15,12 @@
 import pyvista
 from .fileio import from_meshio
 
-POINT_DATA_FIELD = 0
-CELL_DATA_FIELD = 1
-FIELD_DATA_FIELD = 2
-ROW_DATA_FIELD = 6
+
+class FieldAssociation(enum.Enum):
```

Comment:
`codespell` is complaining about this lacking a docstring.  Run `codespell` locally to verify before pushing as we check this on `Azure`.  It's annoying, but it's helped me from pushing garbage code.

Fix this and we're good to go.

Reply:
Ok, I've added a docstring and it's passing all tests here.